src/main/java/org/broadinstitute/hellbender/tools/spark/sv/StructuralVariationDiscoveryPipelineSpark.java

```diff
@@ -40,6 +40,15 @@
             fullName = "contigSAMFile")
     private String outputSAM;
 
+    private static final String OUTPUT_ORDER_SHORT_NAME = "sort";
+    private static final String OUTPUT_ORDER_FULL_NAME = "contigSortOrder";
+
+    @Argument(doc = "sorting order to be used for the output alignment contig SAM/BAM file",
```

Comment:
And the ones in `FindBreakpointEvidenceSpark` as well, so we don't have code duplication.